docs/docs/features/binary-authorization.md

```diff
@@ -277,6 +277,43 @@ informing the user that it was Santa that did it can be a _very_ confusing
 experience for users and lead to wasted time trying to determine the underlying
 cause.
 
+#### CEL
+
+Value: `CEL`
+
+CEL rules allow for more complex policies than would normally be possible. A
+rule with this policy must also include a valid
+[CEL expression](https://cel.dev/) which will be evaluated as part of making a
+decision.
+
+The input to the expression will be an
+[santa.cel.v1.ExecutionContext](https://github.com/northpolesec/protos/blob/704246489aa55e6e2b60b47133a8668bc3656105/cel/v1.proto#L42)
+and the return value must either be a
+[santa.cel.v1.ReturnValue](https://github.com/northpolesec/protos/blob/704246489aa55e6e2b60b47133a8668bc3656105/cel/v1.proto#L20)
+or a bool. If the return value is a bool, true will be treated as a
+ReturnValue.ALLOWLIST and false will be treated as ReturnValue.BLOCKLIST.
+
```

Comment:
We should also call out `target` and it's attributes along with the the helper functions.

Reply:
I linked to the ExecutionContext proto, target is one of the fields in the proto. We don't have any helper functions